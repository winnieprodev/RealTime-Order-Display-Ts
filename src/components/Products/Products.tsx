import React from "react";
import { Row, Col, Table } from 'reactstrap';
import * as Styled from "./products.styled";
import { ProductItem } from "./ProductItem";

const Products = ({ children }: React.PropsWithChildren<{}>) => (
  <Styled.Wrapper>
    <Row>
      <Col xl={12} lg={12} md={12} className="mt-2 mb-2" >
        <Table size="sm" bordered>
          <thead>
            <tr>
              <th className="text-center text-white">No</th>
              <th className="text-center text-white">Customer Name</th>
              <th className="text-center text-white">Cook Name</th>
              <th className="text-center text-white">Destination</th>
              <th className="text-center text-white">Price</th>
              <th className="text-center text-white">Status</th>
            </tr>
          </thead>
          <tbody>
            {children}
          </tbody>
        </Table>
      </Col>
    </Row>
  </Styled.Wrapper>
);

Products.ProductItem = ProductItem;

export default Products;
