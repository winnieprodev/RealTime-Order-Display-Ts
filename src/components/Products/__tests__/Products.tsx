import { render } from "@testing-library/react";

import Products from "../Products";

describe("Products", () => {
  describe("Snapshot tests", () => {
    test("it matches the snapshot for basic use", () => {
      const { container } = render(
        <Products>
          Test
        </Products>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
