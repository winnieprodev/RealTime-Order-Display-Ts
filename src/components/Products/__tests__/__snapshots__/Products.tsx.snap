// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products Snapshot tests it matches the snapshot for basic use 1`] = `
.c0 {
  width: 100%;
}

<div
  class="c0"
>
  <div
    class="row"
  >
    Test
  </div>
</div>
`;
